content/en/docs/setup/install/multicluster/primary-remote_multi-network/index.md

```diff
@@ -242,6 +214,34 @@ $ istioctl x create-remote-secret \
     kubectl apply -f - --context="${CTX_CLUSTER1}"
 {{< /text >}}
 
+## Install the east-west gateway in `cluster2`
```

Comment:
This change isn't necessary, as confirmed by the fact that this is a [tested document](https://github.com/istio/istio.io#testing-document-content). The e/w gateway can be created before or after the cluster is connected. The only difference is that the deployment will not be ready until the cluster is attached to the control plane.

Reply:
Hi Frank,
I had tested official mentioned steps on real cluster with all combination of GKE, EKS, AKS and DO, however, installation of east-west gateway on remote cluster always fails with following error.
<img width="959" alt="image" src="https://user-images.githubusercontent.com/12270920/233776633-c3507142-649f-4fdc-be58-ff0345e8605d.png">

and ReplicaSet of east-west gateway fails with following error.
<img width="956" alt="image" src="https://user-images.githubusercontent.com/12270920/233776686-15048a63-5054-43f2-82d4-9937bab96b08.png">

However, if we deploy remote secret(on primary cluster) before installing east-west gateway on remote cluster, installation goes smoothly and works as expected.

Furthermore, we had been reported issues on similar lines with minikube setup as well, wherein, response was getting received only from primary cluster deployments. And this issue got resolved by applying remote secret before installing east-west gateway on remote cluster.

Tested with:
Istio version: 1.17.1 and 1.17.2
Kubernetes Version: 1.25 and 1.26

Please let me know if I am missing something.

Reply:
Hmm, the 5min timeout error seems to imply that the problem is that you are waiting too long between deploying the gateway and setting the remote secret - i.e., it only work if it's done quickly? If that's the case, maybe it is better to switch the order. Can you confirm that the problem is because there's too much delay between deploying the gateway deploy and the remote secret, but it does work if you do it faster.

Reply:
Could it be related to real cluster vs kind/minikube setup?

Reply:
I have no idea what the explanation is, it really makes no sense.

Anyway, I think your change is probably worth making, regardless, because it doesn't have any expectation that the steps will happen within 5 mins.

You should, however, update the test.sh to change the order that we test the snips, so they are being called in the same order as doc readers will run them.

Reply:
Hi Frank,

I have updated tesh.sh accordingly. Please review.